controllers/manager/monitor.go

```diff
@@ -425,8 +474,27 @@ func ManageStrategy(management CloudManager) bool {
 			log.Info("Vim client is not ready. Wait")
 			return false
 		} else {
+
+			is_existing_strategy, strategy_status, err := IsExistingStrategy(management, client)
+			if err != nil {
+				return false
+			}
+
+			if is_existing_strategy && IsFinishedStrategy(strategy_status) {
+				err = DeleteFinishedStrategy(management, client)
+				if err != nil {
+					return false
+				}
+
+			} else if is_existing_strategy {
+				_ = MonitorExistingStrategy(strategy_status, management)
```

Comment:
The error  of this method is not tracked, if we don't want to track the result, suggest to remove the return value, otherwise need to consider the workflow

Reply:
Done.